packages/core/src/interpreter.ts

```diff
@@ -83,6 +87,8 @@ export class Actor<
   TEvent extends EventObject = EventFromLogic<TLogic>
 > implements ActorRef<TEvent, SnapshotFrom<TLogic>>
 {
+  public static defaults = defaultOptions as Partial<ActorOptions<any>> &
```

Comment:
Why do we expose this as a static property here? It feels like somebody could mutate this to change the default but I would consider that to be a bad thing overall. I wouldn't like a random~ call site to meaningfully change the behavior of all actors in the application. While it might make sense as something convenient in a self-contained system - it quickly breaks apart situations in which actors from different subsystems are put in a single application. A change "intended" for one of them should really not affect **all of them**.

I understand that this might be used to enable devtools globally but I would much prefer a dedicated way of doing this. Inspection is an extra thing that doesn't affect runtime behaviors of all contained subsystems so I think that setting this one thing globally should be fine.

Reply:
What would be the dedicated way of enabling devtools globally that you're thinking of?

Reply:
```ts
import { enableInspection } from 'xstate/devtools'
enableInspection(myInspector)
```

In the future we could even allow this:
```ts
import 'xstate/devtools/enable-browser-devtools'
```

Internally, we would just set a module-level variable and reach for it when instantiating a system.

Reply:
Un-exposed